helpers/prompts.py

```diff
@@ -255,7 +254,10 @@ def prepare_instance_prompt_from_parquet(
         if image_caption:
             image_caption = image_caption.strip()
         if prepend_instance_prompt:
-            image_caption = instance_prompt + " " + image_caption
+            if type(image_caption) == list:
+                image_caption = [instance_prompt + " " + x for x in image_caption]
+            else:
+                image_caption = instance_prompt + " " + image_caption
```

Comment:
Could be nice to have a proper template class that could accept style and subject keywords along with the main body of the caption. So, for example:

<img width="981" alt="image" src="https://github.com/bghira/SimpleTuner/assets/22957388/675231a1-9122-4349-9199-e7b7cc84de0b">

(Figure from https://arxiv.org/abs/2402.12004)

Reply:
i'm not sure if { } can be in a filename, can it?